packages/tutanota-usagetests/lib/model/Stage.ts

```diff
@@ -11,6 +11,19 @@ type MetricConfig = {
 	configValues: Map<string, string>
 }
 
+interface StageCompletionOptions {
```

Comment:
I can't find where it is actually used?

Reply:
I initially used it and thought it would be nice to have but I can remove it.